sacrebleu/metrics/bleu.py

```diff
@@ -62,15 +75,22 @@ def format(self, width=2, score_only=False, signature=''):
             rl=self.ref_len)
         return s
 
+    def __repr__(self):
+        return self.format()
+
 
 class BLEU:
     NGRAM_ORDER = 4
 
     SMOOTH_DEFAULTS = {
-        'floor': 0.0,
+        # The defaults for `floor` and `add-k` are obtained from the following paper
+        # A Systematic Comparison of Smoothing Techniques for Sentence-Level BLEU
+        # Boxing Chen and Colin Cherry
+        # http://aclweb.org/anthology/W14-3346
+        'none': None,   # No value is required
+        'floor': 0.1,
         'add-k': 1,
         'exp': None,    # No value is required
```

Comment:
These, too, should maybe have global constants, so we don't inadvertently use one in only one place?

Reply:
Can't we consider these as global constants since this is a class variable dictionary named `BLEU.SMOOTH_DEFAULTS` ?

Reply:
Oh, sorry, I missed what file this was in (Github cut off the top line showing that this is the actual definition of `SMOOTH_DEFAULTS`)! So ignore my comment here.